src/wallet_cli_main.erl

```diff
@@ -173,22 +175,26 @@ handle_cmd(Specs, Args, OptFun, Fun) ->
             usage(Specs)
     end.
 
-encrypt_keys(Keys=#{public := PubKey}, Password) ->
+encrypt_keys(Keys=#{public := PubKey}, Password, Iterations) ->
```

Comment:
Why not take iterations from Opts?

Reply:
We do?

Reply:
Whoops, sorry, was looking in the wrong spot